Resources/Prototypes/Entities/Objects/Weapons/Guns/Bow/bow.yml

```
@@ -94,3 +94,6 @@
   - type: Construction
     graph: ImprovisedBow
     node: ImprovisedBow
+  - type: Tag
+    tags:
+    - Bow
```

eh........ well well well

```suggestion
  # Starlight
  - type: Tag
    tags:
    - Bow
```